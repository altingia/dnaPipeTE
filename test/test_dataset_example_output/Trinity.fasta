>comp1_c0_seq1
GTCAACAGAGGTGTACACGTTCGATAATCACAGAGCGACTAGTGATATCGACGTGACAAT
CGTCAATGAGGCAGCATCTATGTGGGCCACATATGAGTGGAGAGTGGACGAGTGGGAATT
GAGTGACCACAACATCATTACTGTTGTGGCCGAACCAACTACCGCGCGCGCAGTTGAGAG
CATAGCTCCTGTGCCGTCCTGGAACTTCTCCAATGCACGTTGGCGATTGTTCAAGGAGGA
AATGGTGAGTAGAACAGCCGAACTTCCGGAAAACTTTTCAGAGTCCAACGGC
>comp3_c0_seq1
GTCTCCCACTAAGGGACTATTTTAGGAGGCGGGGAACGATCTCAAGTGACTGACTCATGT
AGTGTGCACTTAAATTACATGTTTTTGAGCAATGCACCCATGTCGCCTTAGATAACAAAA
TCCTAAATATAATTTATCGCTCTCGATTCATTTACATAAGATATGAACGGAGCCCAAAAT
TGTAAGTCTTTAAATATATTCGTGTTCATGTGTGAACAATGGTGGTCTTCATAAATGTCA
TAAATTCCATCATGCTCTGTTGTAAGGCTTGCATCATAGCTTCAGTTTTTGTTTCCTGCT
GCGCAGCTGGGTGATAGTTTTGTTGTGTGTCTAATTTTGTGTGCACTTCATGTGGAGTTC
GGGAATTTTGGGTTGGAGGCATCATACCTGATTTTAAAACGTTAGCA
>comp4_c0_seq1
TTAATATTTATTATGTGTATAAGTGAAAAATGTTGAAATATTCCCATATTCTCTAAGTAT
TATAGAGAAAAGCCATTTTAGTGAATGGATATAGTAGTGTAAGCTAGCTGTTTTACGACA
GAGGGTTCAAAAACTACTATAGGTAGGCAGTGGTTGCCGACCTCTCATATTGTTCAAAAC
GTATGTGTTCATATGATTTTGGCAATTATATGAGTAAATTAAATCATATACATATGAAAA
>comp4_c0_seq2
TTGGGAGGATCTTCGATCACCTGATGCCGCGCTAGTTACATATAAAAGCATTATTTAATA
CAATGACAAAGCCTAGAATCAATTGTAAACGACTTTTGTAACAGGCAAGGTGTTGTAAGT
GGTTGAGCAGCTGCCATACTGCGATCCACTGAAGCTTATCCTTTGCTTGATGATTCGATA
TAAAATAAATGGTTGCCAAACAGCTCGTCATCAATTTAGTGACGCAGGCATATGATATTG
TGTCCCTATCATATAATTTTAATATAAAGAATTTAAAGAATTTTATCAAGAGTAGCCAAA
CACCTCGTCATCAATTTAGTGACGCATATGATATTGTCCCTATCATATAA
>comp5_c0_seq1
ACCGAAATTTGGAAAAACAGACTCTGCAAAAATGTTGATATTTACAAACGAAATTTTCGT
TATAACTTGGCTAAAAATGGTCACATAGATGTAAGAATAACTGTTTTGAGCAGCTAATTA
CCAGTGCTAACGATCCCTATTACTTTTTGAAGGATTTAGGAAATTAATTTTTGGATCAAT
TTTCGCATTTTTTGTAAGGAGGGGGTCATCAAAATTTGCAAAATATGGCCAAAAAATTTA
ATTTCCATTTTTGAACACAGTTTGATTGGAAATTTTATTACGAGCTCAGTGAGGTATGAC
ATTCCATATTCAGACAATTATTTTTTATGTTGTGGCAAAATAAATGATTATTTGATG
>comp6_c0_seq1
GGTCCGAACATAAGTGTGTAAGACAGATTGATAGCTCTTTCTCGAATCTATGGGTGGTGG
TGCATGGCCGTTCTTAGTTCGTGGAGTGATTTGTCTGGTTAATTCCGATAACGAACGAGA
CTCAAATATATTAAATAGATATCTTCAGGATTATGGTGCTGAAGCTTATGTAGCCTTCAT
TCATGTTGGCAGTAAAATGCTTATTGTGTTTGAATGTGTTTATGTAAGTGGAGCCGTACC
TGTTGGTTTGTCCCATTATAAGGACACTAGCTTCTTAAATGGACAAATTGCGTCTAGCAA
TAATGAGATTGAGCAATAACAGGTCTGTGATGCCCTTAGATGTCCTGGGCTGCACGCGCG
CTACAATGAAA
>comp7_c0_seq1
TTAAATGGAACGTGTATCATTTCCTAGCCATTATACGGATATATTTATTATATCTTATGG
TATTGGGTTTTGATGCAAGCTTCTTGATCAAAGTATCACGAGTTTGTTATATAATCGCAA
ACAAATTCTTTAATAAAACGATGCATTTATGTATTTTTGATTTGAAAATTTGGTATAACT
CCAATTACTCAGGTATGATCCAATTCAAGGACATTGCCAGGTAGGGAGTTTGACTGGGGC
GGTACATCTCTCAAATAATAACGGAGGTGTCCCAAGGCCAGCTCAGTGCGGACAGAA
>comp8_c0_seq1
TACAAAATGGATTCATATTATTATATGAAAGAAATATAAAATTTATATATAAATTTGGAA
GAATTATCATGTGCGCTCGGTTTTATGTTATATATTACCAGAGAGTTATATGAAAAGAGA
TAAATTTTAAATTTATCATCAAGATGCAAATGATTTAACTTATATTTGGTTAAACAAAAA
TTGTACAAGTGTGGATACAAAATTTATGTATGTTGGAAATAAAATGATATTTTAGAATGA
AATATATGTATATATAAAGACAAAATTATAGAAAATATATTACAATAATTGTATGATCTT
CTTGTTATATTGGTAAAACAAGTAGAATTTAAAAATGGAAATACGAATTACGAGTGCTAT
ATAAAAATGGCCGTATTCGAATGGATTTATTTTTATAAATATATTTAAAATTTTTACCCA
AAGGCAAAATATTGAATTACATTCAATAATATAAAAAAATGGAATTATATATAAAGTG
>comp9_c0_seq1
CGGAAGGAACCAGCTACTAGATGGTTCGATTGGTCTTTCGCCCCTATACTCAATTCTGAC
AATCGATTTGCACGTCAGAACTGTTTCGGTCTTCCATCAGGGTTTCCCCTGACTTCAACC
TGATCAAGTATAGTTCACCATCTTTCGGGTCACAGCATATATGCTCAAGGTACGTTCCAG
TTAGAGGCATAAATAATATAAATATACATTATACATAACTATATAGAACGCCCCGGGATT
GTGTTAATTAGCTATAAATAGCTAAAAAACTAATCCCATTATTAGTCAAGTTAATTACGC
TATTAGGTTTATATCCCAATAACTTGCACATATGTTAGACTCCTTGGTCCGTGTTTCAAG
ACGGGTCCCGAAGG
>comp10_c0_seq1
TGTTGCGGTTAAAACGTTCGTAGTTGAACTTGTGCTTCATACGGGTAGTACAACTTACAA
TTGTGGTTAGTACTATACCTTTATGTATGTAAGCGTATTACCGGTGGAGTTCTTATATGT
GATTAAATACTTGTATTTTTTCATATGTTCCTCCTATTTAAAAACCTGCATTAGTGCTCT
TAAACGAGTGTTATTGTGGGCCGGTACTATTACTTTGAACAAATTAGAGTGCTTAAAGCA
GGCTTCAAATGCCTGAATATTCTGTGCATGGGATAATGAAATAAGACCTCTGTTCTGCTT
TCATTGGTTTTCAGATCAAGAGGTAATG
>comp11_c0_seq1
GTAGCGAAGCTATGTGCGTGAGGAATTTCATTCCTTCACTAACCGTGATGCCAAGGTACC
GACAGCTACGCACATACGGAAGGTTCGCTCCAGCAAACCTCACCGTAGGCGCACGTCTCA
AGGCACCTTTCAGCAGCATGATTACCGTCTTGCTGGTCGAGACGGCAACGCCAACTTCCG
CTCCCCACGTTTCTACGATGGACATTAGTTGCGCTCCTTTTTCCTCTAGCACAGCTCGGG
AATTTCCCTGGACGAGAAGCAGCAAGTCATCCGCGTATGCACTCAGCTGGCA
>comp12_c0_seq1
CAAAAAATATGAATGAAATATGAAAAGAAAACATTATTCTGGTTGATCCTGCCAGTAGTT
ATATGCTTGTCTCAAAGATTAAGCCATGCATGTCTAAGTACACACGAATTAAAAGTGAAA
CCGCAAAAGGCTCATTATATCAGTTATGGTTCCTTAGATCGTTAACAGTTACTTGGATAA
CTGTGGTAATTCTAGAGCTAATACATGCAATTAAAACATGAACCTTATGGGACATGTGCT
TTTATTAGGCTAAAACCAAGCGATCGCAAG
>comp13_c0_seq1
GTTGGAGGAAGATTGCAAAATTCCAATGCAGAATTTAATGTTAAACATCCAATCATTTTA
GAAAAATGCCACCTAACAAGCTTATTAATAAAAAATGCTCATAAGGAAACATTGCATGGA
GGGATAAACCTAATGCGAAACTATATCCAAAGAAAGTATTGGATTTTCGGGTTGAAAAAT
TCGTTGAAAAAGTATTTAAGAGAATGTGTAACGTGTGCAAGGTATAAACAAAATACAGCT
CAGCAAATAA
>comp14_c0_seq1
CTAAGTTTTACATTATAAGCGGTCAAAATTGGTGATTTTCCGATTTCAAGTACCAGACAA
ACAGAAGATACCTTCTACAGATTATTTAAACCTGGTACACAAAAACAATAAATTGACTAA
GTTATGTCATTTTAAGCGGTCAAAATGGGTGAATTTCCGATTTCAAGTACCAGGCGAACA
GAAGACACCTTCTAGAGATTCTGTTCAACTGGTAAGCAAAAACAGTAAATTGCCTAAGTT
TTACATTTTAAGCGGTCAAAATGGG
>comp15_c0_seq1
AGGTTAGTTTTACCCTACTAATGACAAAACGTTGTTGCGACAGCATTCCTGCGTAGTACG
AGAGGAACCGCAGGTACGGACCAATGGCACAATACTTGTTCGAGCGAACAGTGGTATGAC
GCTACGTCCGTTGGATTATGCCTGAACGCCTCTAAGGTCGTATCCGTGCTGGACTGCAAT
GATAAATAAGGGGCAATTTGCATTGTATGGCTTCTAAACCATTTAAAGTTTATAATTTAC
TTTATAAACGACAATGGATGTGATGCCAATGTAATTTGTAACATAGTAAATTGGG
>comp16_c0_seq1
GAGAAAGGCCAGCTCCTTTAAGCATCTTACAGCTAAAGGTAGCAAAAATAAGTAACTCTT
GTTTCCCCCTACCAAAGGTAAGGAACAGAGTATAAATATAAAAAGCAAAAGATACAAAAG
AATCTTTTATGTTTTAAAACAAGCACCTTATAGTCTATAGCTAAAGGTTGCTTTGTGTAC
CATTATAAATTGTGGTAAGGCGTGCTTGAGGCCATACATCACCAATTGT
>comp17_c0_seq1
TCCCTGCCCTTTGTACACACCGCCCGTCGCTACTACCGATTGAATTATTTAGTGAGGTCT
CCGGACGTGATCACTGTGACGCCTTGCGTGTTACGGTTGTTTCGCAAAAGTTGACCGAAC
TTGATTATTTAGAGGAAGTAAAAGTCGTAACAAGGTTTCCGTAGGTGAACCTGCGGAAGG
ATCATTATTGTATAATATCCTTACCGTTAATAAATATTTGTAATTATACAAATAAAAACA
ATTTACCAAAATAAAAATATAACAAAATGATTCCATGGAATCAAAAGTTAAAATCAAAAT
AAAACGAAGATGGGTTTTATTTATATAGTTAGTGTGGCGCTTGGCAACCTCATAAAAAGA
TTTTAACATTTCTAATGTATG
>comp18_c0_seq1
GAAACTAACAAGGATTTTCTTAGTAGCGGCGAGCGAAAAGAAAACAGTTCAGCACTAAGT
CACTTTGTCTATATGGCAAATGTGAGATGCAGTGTATGGAGCGTCAATATTCTAGTATGA
GAAATTAACGATTTAAGTCCTTCTTAAATGAGGCCATTTACCCATAGAGGGTGCCAGGCC
CGTATAACGTTAATGATTACTAGA
>comp19_c0_seq1
AAATTATTATGTTTATAAGGAATCAATAAAATTCTTTATGATTACGAAACAGCTACACTC
ATTGATTGAAGAAGGCGAAAAAATTCAAGCAGGCATTATAAGCCTGTTGATTGATATTAA
TCACGGTAGGCTAAATACAAATATTCTCAGGCCAAATCAGCTTAAAAAAGAAATTGCCAA
AATTCAGCAGAGTCTTTCGGAGAA
>comp20_c0_seq1
GAACAAAGCTGCTCGCTTATACATTTTTTACCAAACGGATGCCTCAGACATCTTAGGCAA
ATCCCATTTTTTCTTACCCAGTCAGACCGTTCTGCTGGATTCATTATTTTAAATTTATGG
CATTGAATTAAATAATGCCCTGGTAGTTTGCAATATGCACAATTGTCACTATAATTTTTA
TTCTTGTTATTATTAATCATTTTCTTTACAGGT
>comp21_c0_seq1
TGAAACTGCTTAGAGGTTAAGCCCGATGAACCTGAATATCCGTTATGGAAAATTCATCAT
TAAAATTGTAATATTTAAATAATATTATGAGAATAGTGTGCATTTTTTCCATATAAGGAC
ATTGTAATCTATTAGCATATACCAAATTTATCATAAAATATAACTTATAGTTTATTCCAA
TTAAATTGCTTGCATTTTAACACAGAATAAATGTTATTAATTTGATAAAGTGCTGATAGA
TTTATATGATTACAGTGCGTTAATTTATCGGCATTATA
