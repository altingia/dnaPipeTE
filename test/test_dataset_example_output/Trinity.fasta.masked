>comp1_c0_seq1
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNCAACGGC
>comp3_c0_seq1
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNN
>comp4_c0_seq1
TTNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
>comp4_c0_seq2
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNTATAAAATAAATGNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNAA
>comp5_c0_seq1
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNN
>comp6_c0_seq1
GGTCNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNN
>comp7_c0_seq1
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
>comp8_c0_seq1
TACAAAATGGATTCATATTATTATATGAAAGAAATATAAAATTTATATAT
AAATTTGGAAGAATTATCATGTGCGCTCGGTTTTATGTTATATATTACCA
GAGAGTTATATGAAAAGAGATAAATTTTAAATTTATCATCAAGATGCAAA
TGATTTAACTTATATTTGGTTAAACAAAAATTGTACAAGTGTGGATACAA
AATTTATGTATGTTGGAAATAAAATGATATTTTAGAATGAAATATATGTA
TATATAAAGACAAAATTATAGAAAATATATTACAATAATTGTATGATCTT
CTTGTTATATTGGTAAAACAAGTAGAATTTAAAAATGGAAATACGAATTA
CGAGTGCTATATAAAAATGGCCGTATTCGAATGGATTTATTTTTATAAAT
ATATTTAAAATTTTTACCCAAAGGCAAAATATTGAATTACATTCAATAAT
ATAAAAAAATGGAATTATATATAAAGTG
>comp9_c0_seq1
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNN
>comp10_c0_seq1
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNN
>comp11_c0_seq1
GTNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
>comp12_c0_seq1
CAAAAAATATGAATGAAATATGAAAAGNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNN
>comp13_c0_seq1
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
>comp14_c0_seq1
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNATGGG
>comp15_c0_seq1
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
>comp16_c0_seq1
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNN
>comp17_c0_seq1
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNTTGTATAATATCC
TTACCGTTAATAAATATTTGTAATTATACAAATAAAAACAATTTACCAAA
ATAAAAATATAACAAAATGATTCCATGGAATCAAAAGTTAAAATCAAAAT
AAAACGAAGATGGGTTTTATTTATATAGTTAGTGTGGCGCTTGGCAACCT
CATAAAAAGATTTTAACATTTCTAATGTATG
>comp18_c0_seq1
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNN
>comp19_c0_seq1
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNN
>comp20_c0_seq1
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNN
>comp21_c0_seq1
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNN
