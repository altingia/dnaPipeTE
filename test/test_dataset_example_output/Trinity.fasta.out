   SW   perc perc perc  query           position in query    matching     repeat        position in repeat
score   div. del. ins.  sequence        begin end   (left)   repeat       class/family begin  end    (left)  ID

 3009    0.0  0.0  0.0  comp10_c0_seq1      1   328    (0) + SSU-rRNA_Dme rRNA           617    944 (1051)   1  
 2542    1.4  0.3  0.3  comp11_c0_seq1      3   292    (0) C R1_DM        LINE/R1     (1368)   3988   3699   2  
  231    0.0  0.0  0.0  comp12_c0_seq1     28    52  (218) C Jockey-3_DVi LINE/Jockey (5318)     25      1   3 *
 2174    0.4  0.0  0.0  comp12_c0_seq1     36   270    (0) + SSU-rRNA_Dme rRNA             1    235 (1760)   4  
 2225    0.0  0.0  0.0  comp13_c0_seq1      1   250    (0) + ROO_I-int    LTR/Pao       4929   5178 (3078)   5  
  970    8.1  1.1  2.3  comp14_c0_seq1      1   260    (5) + RSP          Satellite        1    257    (0)   6  
 2740    0.0  0.3  0.0  comp15_c0_seq1      1   295    (0) + LSU-rRNA_Dme rRNA          3432   3727  (173)   7  
 2068    0.9  0.0  0.0  comp16_c0_seq1      1   229    (0) + ROO_I-int    LTR/Pao         62    290 (7966)   8  
 1762    0.0  0.0  0.0  comp17_c0_seq1      1   187  (194) + SSU-rRNA_Dme rRNA          1809   1995    (0)   9  
 1479    0.0  0.0 14.0  comp18_c0_seq1      1   204    (0) + LSU-rRNA_Dme rRNA            68    246 (3654)  10  
 1832    0.0  0.0  0.0  comp19_c0_seq1      1   204    (0) + ROO_I-int    LTR/Pao       6818   7021 (1235)  11  
 2608    1.1  0.0  0.0  comp1_c0_seq1       1   285    (7) + R1_DM        LINE/R1       2252   2536 (2820)  12  
 1924    0.0  0.0  0.0  comp20_c0_seq1      1   213    (0) C ROO_I-int    LTR/Pao     (5951)   2305   2093  13  
 2363    0.7  0.0  0.0  comp21_c0_seq1      1   278    (0) + LSU-rRNA_Dme rRNA           398    675 (3225)  14  
 2034    0.0  0.0  0.0  comp3_c0_seq1       1   218  (189) C ROO_LTR      LTR/Pao      (210)    218      1  15  
 1699    1.1  0.0  0.0  comp3_c0_seq1     219   407    (0) C FW_DM        LINE/Jockey (2893)   1799   1611  16  
  673    2.0  3.8  3.8  comp4_c0_seq1       3   107  (133) + NTS_DM       Other          136    240    (0)  17  
 1216    0.0  0.0  0.0  comp4_c0_seq1     108   240    (0) + NTS_DM       Other            1    133  (107)  18  
 1673    0.0  0.0  0.0  comp4_c0_seq2       1   178  (172) + LSU-rRNA_Dme rRNA          3723   3900    (0)  19  
  227   10.0  0.0  0.0  comp4_c0_seq2     192   221  (129) + Jockey-1_DEr LINE/Jockey      2     31 (4678)  20 *
  294    8.4  3.5 15.8  comp4_c0_seq2     215   299   (51) + Jockey-1_DEr LINE/Jockey   4624   4699   (10)  21  
  266   19.6  0.0  2.0  comp4_c0_seq2     280   331   (19) + LSU-rRNA_Dme rRNA          2570   2620 (1280)  22 *
  306    0.0  0.0  0.0  comp4_c0_seq2     315   348    (2) + Jockey-1_DEr LINE/Jockey   4624   4657   (52)  23  
 1834   16.1  1.1  0.6  comp5_c0_seq1       1   357    (0) C SAR_DM       Satellite    (542)    930    572  24  
 3418    0.0  0.0  0.0  comp6_c0_seq1       5   371    (0) + SSU-rRNA_Dme rRNA          1303   1669  (326)  25  
 2748    0.0  0.0  0.0  comp7_c0_seq1       1   297    (0) + LSU-rRNA_Dme rRNA          2845   3141  (759)  26  
 3503    0.0  0.0  0.0  comp9_c0_seq1       1   374    (0) C LSU-rRNA_Dme rRNA        (2775)   1125    752  27  
